injection/sharedmain/main.go

```diff
@@ -167,13 +167,18 @@ func MainWithConfig(ctx context.Context, component string, cfg *rest.Config, cto
 	if _, err := kubeclient.Get(ctx).CoreV1().ConfigMaps(system.Namespace()).Get(logging.ConfigMapName(),
 		metav1.GetOptions{}); err == nil {
 		cmw.Watch(logging.ConfigMapName(), logging.UpdateLevelFromConfigMap(logger, atomicLevel, component))
+	} else if !apierrors.IsNotFound(err) {
```

Comment:
Do we not care if it is NF?

Reply:
If it is not found, we just bypass watching it in cmw. I do not think that case should block the process.

Reply:
I'd log it, rather than kill, but OK.